spec/mem/tree_borrows/memory.md

```diff
@@ -90,8 +90,7 @@ impl<T: Target> TreeBorrowsMemory<T> {
         frame_extra: &mut TreeBorrowsFrameExtra,
     ) -> Result<Pointer<TreeBorrowsProvenance>> {
         let Some((alloc_id, parent_path)) = ptr.provenance else {
-            // An invalid pointer.
-            throw_ub!("Tree Borrows: reborrowing pointer without provenance");
+            return ret(ptr);
```

Comment:
```suggestion
            // If the pointer has no provenance, just keep it unchanged. It can't be used to access any memory anyway.
            return ret(ptr);
```

Reply:
Hm wait... this is only okay if `pointee_size` is zero. If it is non-zero, this should be UB!

Reply:
oops, I added a UB test case for this...

https://github.com/minirust/minirust/pull/221/files#diff-14483848f1f65ed137acedd3856e275ef52b1b2516bdd8e84424941890f6c7ba